pilot/helpers/cli.py

```diff
@@ -112,6 +112,35 @@ def terminate_process(pid: int, name=None) -> None:
            del running_processes[command_id]
 
 
+def read_queue_line(q, stdout=True):
+    try:
+        line = q.get_nowait()
+    except queue.Empty:
+        line = None
```

Comment:
Looks we can just `return ""` here, and we don't need the `if line` checks below?

Reply:
oh yeah, nice catch! Thanks